src/data-types.ts

```diff
@@ -0,0 +1,9 @@
+export * from './dialects/abstract/data-types';
+export * as db2 from './dialects/db2/data-types';
```

Comment:
what is the reason for exposing these? Shouldn't users only use `DataTypes.x` and not `DataTypes.postgres.x`?